plugins/utils/which-key.nix

```diff
@@ -247,19 +247,25 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
         Please refer to the plugin's [documentation](https://github.com/folke/which-key.nvim?tab=readme-ov-file#%EF%B8%8F-mappings).
       '';
       example = [
-        {
-          __unkeyed-1 = "<leader>w";
-          proxy = "<C-w>";
-          group = "windows";
-        }
```

Comment:
Is there an issue with having a `proxy` mapping example?

Reply:
Sorry, just deleted a group of them and didn't notice the proxy example.